### PR TITLE
Fix unpacking configs into slices

### PR DESCRIPTION
**Pull request by urso**
Fix unpacking into top-level slice/array or unpacking into slice if
`inline`-tag is used.

### CODE REVIEW

**Comment by urso** [comment]
Closes #45

File: reify.go [hunk 1]
```diff
@@ -17,7 +17,10 @@ func (c *Config) Unpack(to interface{}, options ...Option) error {
 	}
 
 	vTo := reflect.ValueOf(to)
-	if to == nil || (vTo.Kind() != reflect.Ptr && vTo.Kind() != reflect.Map) {
+
+	k := vTo.Kind()
+	isValid := to != nil && (k == reflect.Ptr || k == reflect.Map)
```
Comment (ruflin) [inline reify.go]: That lines is quite hard to read

**Comment by ruflin** [comment]
LGTM

File: reify.go (hunk 1 above)
Comment (urso) [inline reify.go]: I think it's easier to read then before. it says: to must not be nil and must be either a pointer or a map

**Pull request merged**